modules/mono/mono_gd/gd_mono_marshal.cpp

```diff
@@ -1536,4 +1736,5 @@ M_SignalInfo signal_info_to_managed(const Signal &p_signal) {
 	MonoObject *name_string_name_managed = GDMonoUtils::create_managed_from(p_signal.get_name());
 	return { owner_managed, name_string_name_managed };
 }
+
```

Comment:
The opinionated `file_format.sh` doesn't like this line.
It's because we're trying to enforce
```cpp
if (cond) {
	some_code;
}
```
over
```cpp
if (cond) {

	some_code;
}
```
or
```cpp
if (cond) {
	some_code;

}
```

But it's not clever enough to be more lenient with namespaces.

Reply:
Fixed!